sales.html

```diff
@@ -7,13 +7,30 @@
         <meta name="viewport" content="width=device-width, initial-scale=1.0">
     </head>
     <body>
-        <h1>Salmon Cookies: Hourly Sales Projections</h1>
-        <table>
-            <caption>Cookies Needed By Location Each Day</caption>
-            <thead></thead>
-            <tbody></tbody>
-            <tfoot></tfoot>
-        </table>
+        <article>
+            <h1>Salmon Cookies: Hourly Sales Projections</h1>
+            <table>
+                <caption>Cookies Needed By Location Each Day</caption>
+                <thead></thead>
+                <tbody></tbody>
+                <tfoot></tfoot>
+            </table>
+        </article>
+        <article>
+            <h2>New Store Input</h2>
+            <form>
+                <label for="store-name">Store Name</label>
+                <input type="text" id="store-name">
```

Comment:
Good work here not including a self-closing tag or a closing tag at all. It looks like [void elements aren't supposed to have any kind of closer](https://www.w3.org/TR/2011/WD-html-markup-20110113/syntax.html#:~:text=A%20void%20element%20is%20an,param%20%2C%20source%20%2C%20track%20%2C%20wbr#T).